lib/ts/recipe/session/index.ts

```diff
@@ -82,7 +82,7 @@ export default class SessionWrapper {
     }
 
     static createJWT(payload?: any, validitySeconds?: number) {
-        let jwtRecipe = Recipe.getInstanceOrThrowError().jwtRecipe;
+        let jwtRecipe = Recipe.getInstanceOrThrowError().openIdRecipe?.jwtRecipe;
```

Comment:
Do not use jwtRecipe anywhere in this file. Only use openIdRecipe

Reply:
Done